web/frontend/src/lib/error.ts

```diff
@@ -1,9 +1,17 @@
 import type { ServiceError } from '@viamrobotics/sdk';
 import { toast } from './toast';
 
-export const displayError = (error: ServiceError | null) => {
-  if (error) {
-    toast.error(error.message);
+export const isServiceError = (error: unknown): boolean => {
+  return error instanceof Object && error && 'message' in error && 'code' in error && 'metadata' in error
+};
+
+export const displayError = (error: ServiceError | string | null) => {
```

Comment:
I updated this so that we are able to display errors of strings in addition to ServiceErrors